test/fika/type_checker_test.exs

```diff
@@ -14,6 +14,22 @@ defmodule Fika.TypeCheckerTest do
     assert {:ok, "Int", _} = TypeChecker.infer_exp(Env.init(), ast)
   end
 
+  test "infer type of atom expressions" do
+    str = ":a"
+
+    {:ok, [ast], _, _, _, _} = Fika.Parser.expression(str)
```

Comment:
We can use `ast = Fika.Parser.expression!(str)` here (and also in the test below)